benchmarks/mempool-long-lived.py

```diff
@@ -90,9 +91,15 @@ async def get_coin_record(coin_ids: Collection[bytes32]) -> list[CoinRecord]:
                 ret.append(r)
         return ret
 
+    # We currently don't need to keep track of these for our purpose
+    async def get_unspent_lineage_info_for_puzzle_hash(_: bytes32) -> Optional[UnspentLineageInfo]:
```

Comment:
just curious why you didn't also name this `zero_calls_...` like `test_mempool_manager`

Reply:
Hmm, I guess in consistency with `get_coin_record` right above it, considering this is not production code/tests, I just went with that. Notice I did the same thing with the one in `benchmarks/mempool.py` too.